test/e2e/handler/utils.go

```diff
@@ -61,22 +61,26 @@ func interfaceByName(interfaces []interface{}, searchedName string) map[string]i
 	return dummy
 }
 
-func setDesiredStateWithPolicyAndNodeSelector(name string, desiredState nmstate.State, nodeSelector map[string]string) {
+func updateNodeNetworkConfigurationPolicy(name string, desiredState nmstate.State, nodeSelector map[string]string) error {
```

Comment:
Let's rename this `setDesiredStateWithPolicyAndNodeSelector`.

Reply:
done